### ISSUE TITLE
Introducing: Colmena

### ISSUE THREAD

**Issue by beeman** [colmena/colmena#204]
## 🍯 Colmena

~I have been working on a new version of this project and as of today it will be [available in a branch](https://github.com/beeman/loopback-angular-admin/tree/colmena-cms) in this project.~

~Once most of the functionality of the `v0.x` project is in place I will archive the current master and rename the github project to `colmena-cms` and move it to the [colmena](https://github.com/colmena) organization.~

To concentrate the development effort on the new version I have closed all outstanding issues, most of theme where inactive anyway due to the small movement in the project due to a lack of time.

## Versions
The admin interface of Colmena will be using the latest stable version of Angular (4.x at this moment) and aims to be updated. ~The backend will be LoopBack v2 as a lot of the components and mixins are not yet available for v3.~ The backend is now LoopBack v3.

## Timeline
While it's hard to give an exact timeline I will aim to get most the functionality working within 2 weeks from now (February 1st 2017), the progress can be followed here.

## Contributions
Contributions are obviously welcome. As there are currently a lot of moving parts, it probably makes sense to first discuss what you'd like to contribute in order to don't waste time working on things that are not needed/are already being worked on. To get in touch please create an issue or [join me on the Colmena slack](https://colmena-slack.now.sh/).

## Tracking progress
- [ ] [Build out User Authentication](https://github.com/beeman/loopback-angular-admin/issues/208)
- [ ] [Build out basic Content Management](https://github.com/beeman/loopback-angular-admin/issues/207)
- [ ] [Add multi-language support](https://github.com/beeman/loopback-angular-admin/issues/206)
- [ ] [Create documentation](https://github.com/beeman/loopback-angular-admin/issues/209)
- [x] [Rename project and move to colmena org](https://github.com/beeman/loopback-angular-admin/issues/205)

**Comment by denzelwamburu** [comment]
:+1: 
Am excited about this new project.

**Comment by beeman** [comment]
Closing this story as it's no longer relevant.
